### PR TITLE
documentation / minor fixes

### PR DESCRIPTION
**Pull request by OriHoch**
* added CONTRIBUTING.md file - shamelessly copied from Angular project with some modifications
* added .nvmrc file - to specify node version to use for the project (I used latest 7.4.0, but maybe you used a different version)
* added document for setting up dev environment from scratch
* updated some of the sql files to allow creating a DB from scratch
* removed IDE configuration files - they are very user specific and keep having local changes. If needed - can include a docs/ide_configurations directory with suggested configurations

### CODE REVIEW

**Comment by funkycode** [comment]
regarding "removed IDE configuration files - they are very user specific and keep having local changes. If needed - can include a docs/ide_configurations directory with suggested configurations"
Shouldn't it be added to git_ignore and pushed to repo so ti would not be commited back by someone else?

**Comment by OriHoch** [comment]
@funkycode - if it's in .gitignore - it will not be committed back by someone else

keeping the directory committed but having it in .gitignore can cause problems in my experience..

**Comment by natekonimbo** [comment]
@OriHoch well done!
I suggest code and documentation changes separation into different branches for sanity. what do you think?

**Comment by OriHoch** [comment]
sure @natekonimbo  - I'll take out the code changes to a new PR

**Comment by OriHoch** [comment]
done, moved code changes to #11

File: sql/schema.sql [hunk 1]
```diff
@@ -32,9 +35,10 @@ CREATE TABLE IF NOT EXISTS users (
   facebook_token            VARCHAR(255),
 
   # Camp relations
-  camp_id                   INTEGER,
+  camp_id                   INTEGER-- ,
```
Comment (netanelben) [inline sql/schema.sql:38]: @OriHoch comment ?

**Comment by netanelben** [comment]
- Agree with @OriHoch to remove ``.idea`` folder.
- .sql files :ok:
- .md files :ok:

File: sql/schema.sql (hunk 1 above)
Comment (OriHoch) [reply sql/schema.sql:38]: yes, because I removed the field after it..

**Pull request merged**